department-of-veterans-affairs/va.gov-cms
# VACMS-8467: Update button mixin to accept arguments.

## Description
Closes #8467 .
 
## Testing done
Edited a node and navigated to an audit view and confirmed button styles were still functional.

## Screenshots
![image](https://user-images.githubusercontent.com/8375335/160416303-1a53725b-2e90-4640-87fb-fe8b63c4f1a3.png)
![image](https://user-images.githubusercontent.com/8375335/160416324-005ff02c-85fe-4ba3-9380-e3f89ffff2d0.png)


## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As CMS user
1. Edit a node (`/node/35055/edit`) and open the rich text editor
   - [x] Validate that button styles still work as expected
2. Then view the node and
   - [x] Validate that button styles still work as expected
3. Navigate to a view (`/admin/content/media/pdfs`) and
   - [x] Validate that button styles still work as expected
4. Then validate Acceptance Criteria from issue
   - [x] Mixin is updated to include arguments for background and background hover
   - [x] Button styles are unchanged on the frontend of the CMS

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
